target/linux/mvebu/wdmc/base-files/lib/preinit/05_wdmc_leds

```diff
@@ -0,0 +1,20 @@
+preinit_leds_init() {
+
+	. /lib/functions.sh
+	. /lib/functions/leds.sh
+
+	case $(board_name) in
+	armada-385-wd-mcex2u)
+		[ -e /dev/sda ] && {
+			echo "HDD1 (sda) is online, LED trigger ata1" > /dev/kmsg
+			led_set_attr "wdmcex2u:blue:hdd1" "trigger" "ata1"
```

Comment:
This LED triggers should be set in the DTS via `linux,default-trigger = "ata1";` property. Well, ideally with upstream supported trigger `linux,default-trigger = "disk-activity";` but AFAIK this trigger currently doesn't support different/selectable ports.

Reply:
It's a bit tricky here. These blue LEDs are used as an activity as well as disk presence/healthy state indicators. They are inverted. So just defining ataX in DTS is not enough, LEDs will always be ON, even for absent drives. Also, as they share physical space with the red ones (not inverted), they should be turned off when we want indicate an error (like SMART failure, feature that I'm still working on).

Yes, disk-activity is only suitable for a single HDD LED at this moment, hence is all that ARCH_WANT_LIBATA_LEDS jumping around.